File: my-timer/src/main/java/com/example/mytimer/DemoApplication.java
package com.example.mytimer;

import com.example.mytimer.demo.QuartzSchedulerDemo;
import com.example.mytimer.demo.RedisReadWriteService;
import org.quartz.SchedulerException;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ConfigurableApplicationContext;

import java.util.concurrent.TimeUnit;

@SpringBootApplication
//@EnableScheduling
public class DemoApplication {

    private static final Logger LOGGER = LoggerFactory.getLogger(DemoApplication.class);


	public static void main(String[] args) {
        ConfigurableApplicationContext context = SpringApplication.run(DemoApplication.class, args);

        QuartzSchedulerDemo quartzSchedulerDemo = context.getBean(QuartzSchedulerDemo.class);

        RedisReadWriteService redisReadWriteService = context.getBean(RedisReadWriteService.class);
        redisReadWriteService.save();
        LOGGER.info(quartzSchedulerDemo.toString());

        String quartzJobName = "mydemo";
        try {
            quartzSchedulerDemo.start(quartzJobName, "0/3 * * * * ?");
        } catch (SchedulerException e) {
            LOGGER.error("SchedulerException : ", e);
        }


        //TimerDemo.execute();

        try {
            TimeUnit.MINUTES.sleep(1L);
        } catch (InterruptedException e) {
            LOGGER.warn("SchedulerException : ", e);
            Thread.currentThread().interrupt();
        }

        String v = redisReadWriteService.read();
        System.out.println(" v ============> "+ v);
        quartzSchedulerDemo.stop(quartzJobName);
    }
}

File: group-wechat/wechat-backend-001/src/main/scala/com/example/demo/DemoApplication.scala
package com.example.demo

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

/**
  * <p>
  * <strong>
  * 用一句话描述功能
  * </strong><br /><br />
  * 如题。
  * </p>
  *
  * @author chengchaos[as]Administrator - 2019/7/8 10:42 <br />
  * @see 【相关类方法】
  * @since 1.1.0
  */
@SpringBootApplication
class DemoApplication

object DemoApplication {

  def main(args: Array[String]): Unit = {
    SpringApplication.run(classOf[DemoApplication], args:_*)
  }

}
